### DISCUSSION TITLE
Is VizTracer enable its support for multiProcessing.Pool.Map?

### DISCUSSION THREAD

**Discussion by xunyi0320** [gaogaotiantian/viztracer#438] [Q&A] [answered]
I want to compare performance between multiprocessing.Process and multiprocessing.pool.map, however, it doesn't work on map.It doesn't work, like this. It doesn't show times in single processer. I find only pool.async_apply method, but not pool.map.Is there some ways?

我想对比一下自己新建Process和使用Pool，然后在里面应用map的比较，但是使用VT好像如下图所示没有显示出来每一个进程的运行时间等信息，我只在文档里面找到了Pool async_apply 方法的，没找到map的，这里有什么其他方法或者参数吗（当然我也没在文档里面找到）

![image](https://github.com/gaogaotiantian/viztracer/assets/75795033/fe1d7aae-4cd1-488a-a858-580664570683)
with code 

```
from utils.is_prime import is_prime
import time
import multiprocessing as mp
if __name__ == '__main__':
    start_time = time.time()
    MAX_N = 1000000
    with mp.Pool(10) as pool:
        result = pool.map(is_prime,range(2,MAX_N),MAX_N//16)
    cnt = result.count(True)
    print(cnt)
    print(time.time()-start_time,"s")
```

where is_prime function: 其中is_prime 函数
```
def is_prime (number:int):
    sqrt_number = math.sqrt(number)
    for i in range(2,int(sqrt_number)+1):
        if (number % i ) == 0:
            return False
    return True
```

**Comment by gaogaotiantian** [comment, marked as answer]
Are you using Windows? It's impossible (well not with some serious hacks to the stdlib) to support pools on Windows because Windows terminates the process workers abruptly without giving viztracer any change to intercept.

**Reply by xunyi0320** [reply]
高天兄最帅了~（来自b站粉丝的爱），没错我用的windows，使用linux就正常了，把这个问题写进文档里或者其他什么地方怎么样。毕竟用windows的人也挺多的
gaogaotiantian is handsome!!!!!!!!!(love from a fan from bilibili). I was using windows, and now I turn to linux with a awsome instance of viztracer!How about record it into document or somewhere else? I think some many may have same problem with Windows. Too many people use Windows.

![image](https://github.com/gaogaotiantian/viztracer/assets/75795033/fc05023c-44e7-4fc2-b17f-8cf5662a597e)

**Reply by xunyi0320** [reply]
为什么我写的中文一股翻译腔？英文也不正宗，完蛋力（悲
